### DISCUSSION TITLE
So It was a paid service? What a pity :( , But thanks for your HamsterCode

### DISCUSSION THREAD

**Discussion by yanpaing007** [masterking32/MasterCryptoFarmBot#7] [General]

**Comment by marrsripon** [comment]
great work always need to appreciate . We are ready to donate . He is best in this sector

**Comment by AmirhN1382** [comment]
Some bots will be free, some paid. Still to be decided, there might be even free version of paid ones if possible with less features.
But don't worry, famous bots will be probably free as always to attract people. For example you don't see anyone selling HamsterKombat bots, but the other bots that are not as famous as HamsterKombat are mostly being sold.
